pkgs/development/python-modules/colormath2/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  networkx,
+  numpy,
+  pytestCheckHook,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "colormath2";
+  version = "3.0.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "bkmgit";
+    repo = "python-colormath2";
+    rev = "${version}";
+    hash = "sha256-G8b0L8A2RzbVQFPNg2fuBklqTNjo3yqvek/+GnqtsHc=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    networkx
+    numpy
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "colormath2" ];
+
+  meta = with lib; {
```

Comment:
using `with lib;` is now discouraged as it is shown to increase eval times. Please remove it.